R-TypeGame/Components/Shoot/ShootComponent.hpp

```diff
@@ -0,0 +1,30 @@
+/*
+** EPITECH PROJECT, 2024
+** R-Type game
+** File description:
+** ShootComponent
+*/
+
+#pragma once
+
+#include "../../../Engine/Shared/Interface/IComponent.hpp"
+
+/**
+ * @brief Component Shoot for R-Type Game.
+ *
+ */
+class ShootComponent : public AComponent {
+
+    public:
+
+        int damage;         // Damage of one bullet.
```

Comment:
Can damage be negative ?

Reply:
Perhaps if you enable healing with friendly shots ?

Reply:
What a weird feature but okay 😂

Reply:
No it's not, imagine a boost with which you can heal your friends during a short time.

Reply:
Yeah, why not, but I think that this feature should be implemented in the system that checks iff the shot hits an enemy or an ally, then apply `damage` to enemy or heal allies with `damage`

Reply:
But I'm fine with keeping damage as an integer 👍